### DISCUSSION TITLE
ACCES_BACKGROUND_LOCATION by Shiny.Locations?

### DISCUSSION THREAD

**Discussion by bodeg** [shinyorg/shiny#717] [Q&A] [answered]
I used useGps() in my Xamarin Forms project, and published it to Google store.

My Gps request for StartListener is: 

`GpsRequest.Realtime(backgroundMode: false)`

Google rejected the app because the app wants ACCES_BACKGROUND_LOCATION permission.
But I do NOT have it in my manifest. 
Is it come because I added Shiny.Locations and it was added during build?

if so...how can it be solved?

With regards, alon/

**Comment by aritchie** [comment, marked as answer]
Shiny GPS is really meant for background operations.  It has a code injection for the manifest which is creating this.  I'll consider doing this in the future, but for now it is not a priority as it is easier to have this automatically for now.

**Comment by bodeg** [comment]
Thank you very much for the prompt reply.
